### PR TITLE
Threshold and detail enpoint

### PR DESCRIPTION
**Pull request by sara-02**
> Waiting for the PR builds to pass :confused:

> Builds are passing now, but not picking the right image name :confused: 

All good now :tada:

### CODE REVIEW

**Comment by sara-02** [comment]
[test]

**Comment by sara-02** [comment]
[test]

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,31 @@
+ifeq ($(TARGET),rhel)
+  DOCKERFILE := Dockerfile.rhel
```
Comment (rootAvish) [inline Makefile]: Your repository does not contain a Dockerfile.rhel, and neither does this PR. You need to create that.

Comment (sara-02) [reply Makefile]: @rootAvish For testing DockerRHEL I added that in another PR https://github.com/fabric8-analytics/f8a-hpf-insights/pull/16

Comment (rootAvish) [reply Makefile]: It needs to be merged before #15, RHEL builds are not done for PRs.

Comment (sara-02) [reply Makefile]: @rootAvish #16 is merged now.

**Comment by rootAvish** [comment]
@sara-02 Feel free to merge this yourself when you're ready.

**Comment by sara-02** [comment]
Thanks, merging this now.

**Pull request merged**